Userland/DevTools/HackStudio/Project.cpp

```diff
@@ -59,4 +59,9 @@ String Project::to_absolute_path(String const& path) const
     return LexicalPath { String::formatted("{}/{}", m_root_path, path) }.string();
 }
 
+bool Project::project_is_serenity() const
+{
+    return m_root_path.ends_with("Source/serenity");
```

Comment:
This heuristic is a little bit weird, but all right :^)

Reply:
It is yeah. Added a comment & a FIXME for it.